service/utils/utils.go

```diff
@@ -0,0 +1,47 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
Definitively that utils package deserves another name, though I don't know which one, tbh